### PR TITLE
Added privacy policy

### PR DESCRIPTION
**Pull request by ChrisMeyer7088**
# Description

Added privacy policy

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manually

# Notes

Just added privacy policy route along with privacy policy vue. Added another menu item and made the drop down unable to appear if the user is logged in

### CODE REVIEW

File: Frontend/src/components/Navbar.vue [hunk 1]
```diff
@@ -3,7 +3,8 @@
     <v-menu  
         bottom 
         offset-y
-        transition="slide-x-transition">
+        transition="slide-x-transition"
+        v-if="stored.state.isLogin">
```
Comment (julianpoy) [inline Frontend/src/components/Navbar.vue:7]: Why this change?

**Review by julianpoy — APPROVED** [review]

File: Frontend/src/components/Navbar.vue (hunk 1 above)
Comment (ChrisMeyer7088) [reply Frontend/src/components/Navbar.vue:7]: Privacy policy should be accessible by anyone, only logged in users should see our features like map view and point editor

Comment (julianpoy) [reply Frontend/src/components/Navbar.vue:7]: Coolio
